README.md

```diff
@@ -14,7 +14,6 @@ As the resources deployed need cluster-admin privileges, credentials of a super
 #### Deploy from the UI
```

Comment:
This typo `s/UI/CLI/` is like disease, spreading across projects  :-D ...

Reply:
you made me laugh